syz-fuzzer/proc.go

```diff
@@ -276,17 +276,17 @@ func (proc *Proc) enqueueCallTriage(p *prog.Prog, flags ProgTypes, callIndex int
 	})
 }
 
-func (proc *Proc) executeRaw(opts *ipc.ExecOpts, p *prog.Prog, stat Stat, enableDF Stat, disableDF Stat) *ipc.ProgInfo {
+func (proc *Proc) executeRaw(opts *ipc.ExecOpts, p *prog.Prog, stat Stat) *ipc.ProgInfo {
 	if opts.Flags&ipc.FlagDedupCover == 0 {
 		log.Fatalf("dedup cover is not enabled")
 	}
 	dfDisable := prog.DFetchAnalysis(p)
 	if dfDisable {
-		atomic.AddUint64(&proc.fuzzer.stats[disableDF], 1)
+		atomic.AddUint64(&proc.fuzzer.stats[StatEnableDF], 1)
```

Comment:
Please fix: should be StatDisableDF

Reply:
Or maybe the opposite...
Why are we counting this program as disabled, but set the flag?

Reply:
Ok, I get it. Despite the flag is called 'FlagDoubleFetch', executor treats it as 'disable double fetch detection'.

I don't think this is a good idea: by default the executor must only enable the very basic functionality, everything else should be explicitly requested via flags.
So let's change this code to pass the flag when DF detection is to be enabled, and let the executor do:
```
  if (flag_df)
    df_enable();
```

Reply:
okay i will change it